css/_base.scss

```diff
@@ -51,6 +51,22 @@
             ol {
               margin-left: 20px;
             }
+
+            a {
+              color:teal;
+            }
+
+            a:hover {
+              color: lighten(teal, 15%);
+              -o-transition: color .2s;
+              -ms-transition: color .2s;
+              -moz-transition: color .2s;
+              -webkit-transition: color .2s;
+            }
+
+            a:visited {
```

Comment:
Remove a:visited, since we want the link to always be the normal teal color unless hovered on :)